packages/nx/src/project-graph/utils/project-configuration-utils.spec.ts

```diff
@@ -5,12 +5,14 @@ import {
 } from '../../config/workspace-json-project-json';
 import {
   ConfigurationSourceMaps,
+  createProjectConfigurations,
   isCompatibleTarget,
   mergeProjectConfigurationIntoRootMap,
   mergeTargetConfigurations,
   readProjectConfigurationsFromRootMap,
   readTargetDefaultsForTarget,
 } from './project-configuration-utils';
+import { CreateNodes, NxPluginV2 } from 'nx/src/utils/nx-plugin';
```

Comment:
ci failed due to lint. could you change this import to a relative path?

Reply:
Indeed! Just noticed and fixed it 😊.
Still surprised that WebStorm didn't prefer the relative path 🤔